app/assets/stylesheets/report.scss

```diff
@@ -32,3 +45,13 @@
     text-align: center;
   }
 }
+
+.rlabel {
+  display: inline-block;
```

Comment:
Properties should be ordered border-right, display, font-weight, margin-right, padding-right, text-align, width